api.py

```diff
@@ -3,7 +3,7 @@
 def hi():
     print('hiii')
 
-def ocr_space_file(filename, overlay=True, api_key='0fa8a9fbf988957', language='eng'):
+def ocr_space_file(filename, overlay=True, api_key='enter_your_key', language='eng'):
```

Comment:
change your api key as it is exposed already

Reply:
ok